torchmultimodal/utils/common.py

```diff
@@ -20,6 +21,50 @@ def get_current_device():
         return torch.device("cpu")
 
 
+def shift_dim(
```

Comment:
Even though we might have already hit the code path in `test_preprocess` for the codebook, it's better to have a dedicated test for this utility.